.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/home/eyoung85/miniconda3/envs/sourmash_dev/bin/python"
+}
```

Comment:
Can you remove this file from the pull request, please? :) It's a local config file for your VSCode and shouldn't be in the git repo.

If you do:
```
git rm --cached .vscode/settings.json
``
it will remove it from git but not from your local directory. Then `git commit`.

If you like, you can add `.vscode` to the `.gitignore` file and git will ignore it from then on.